### PR TITLE
Issue 88 teaser export

### LINKED ISSUE #88: TEASER export
Allow an easy export of TEASER buildings into BESMod [TEASER Issue 771](https://github.com/RWTH-EBC/TEASER/issues/771).
- [ ] Add partial example system models which are extended in the TEASER export as ready to run models with the specific TEASER building for the calculation of heat loads and monoenergetic energy usages.
- [ ] Allow easy export and usage of retrofit cases from TEASER in BESMod

### PR DESCRIPTION
**Pull request by HvanderStok**
Closes #88.

This pull request adds partial models for the [TEASER](https://github.com/RWTH-EBC/TEASER) export and an example of it for regression testing.

Additionally, it fixes some bugs related to the simulation of multiple zones and the `AutoparameterBoiler`.
The automatic sizing is now done with the nominal firing power instead of the nominal heat flow, and new regressions of pressure drop and volume for different power classes of boilers have been added. Also, the changes to the boiler in the [AixLib](https://github.com/RWTH-EBC/AixLib/pull/1543) are implemented and the used AixLib version is updated.

The default partial retrofit option of the radiator systems is changed so, that these systems are not retrofitted if `QBuiOld_flow_design` is set and different from `QBui_flow_nominal`. 

In the process, partial retrofit options were implemented to the models `Systems.Hydraulical.Generation.GasBoiler` and `Systems.Hydraulical.Distribution.DistributionTwoStorageParallel` which are not necessarily needed any more in this pull request and could be reverted.

### CODE REVIEW

File: BESMod/unitTests-dymola.log [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Regression tests are only run for the following package:
```
Comment (FWuellhorst) [inline BESMod/unitTests-dymola.log:1]: When was this added?

File: BESMod/Systems/Hydraulical/Transfer/RadiatorPressureBased.mo [hunk 2]
```diff
@@ -10,7 +10,7 @@ model RadiatorPressureBased "Pressure Based transfer system"
     Q_flow_design={if use_oldRad_design[i] then QOld_flow_design[i] else Q_flow_nominal[i] for i in 1:nParallelDem},
     TTra_design={if use_oldRad_design[i] then TTraOld_design[i] else TTra_nominal[i] for i in 1:nParallelDem});
 
-  parameter Boolean use_oldRad_design[nParallelDem]=fill(false, nParallelDem)
+  parameter Boolean use_oldRad_design[nParallelDem]={if QOld_flow_design[i]==Q_flow_nominal[i] then false else true for i in 1:nParallelDem}
     "If true, radiator design of the building with no retrofit (old state) is used"
```
Comment (FWuellhorst) [inline BESMod/Systems/Hydraulical/Transfer/RadiatorPressureBased.mo:14]: Is this necessary? If they are the same, Q_flow_nominal will be used. If not, false leads to Q_flow_nominal being used anyway. So in both cases _nominal is used?

File: BESMod/Systems/Hydraulical/Generation/RecordsCollection/AutoparameterBoiler.mo [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
+    a=if Q_nom<=72250 then -103783 * Q_nom + 1E+10 elseif (72250<Q_nom and Q_nom<165750) then 2.76E8 else 9.1132E7,
+    n=2,
     Q_min=Q_nom * 0.3,
     eta=[0.3,0.93; 1.0,0.93]);
   annotation (Documentation(info="<html>
-<p>All Vitodens boilers have the same Q_min of 30 &percnt; of Q_nom. A linear regression was used to estimate the pressure drop (R2=0,9315) and volume (R2=0,9751). </p>
+<p>Automated sizing based on Vitogas 200-F (11-60kW and 72-144kW) and Vitocrossal 200 (82-311kW). </p>
+<p>These boilers have the same Q_min of 30 &percnt; of Q_nom. 
+The pressure drop coefficent was estimatet with a linear regression for small powers and constant values for medium and high power boliers.
+The volume was estimated with a linear regression over the total power range. (See AixLib/Resources/AutoparameterBoiler.xlsx)
```
Comment (FWuellhorst) [inline BESMod/Systems/Hydraulical/Generation/RecordsCollection/AutoparameterBoiler.mo:14]: See BESMod\
and, could you move the excel to `data` folder?

**Review by FWuellhorst — COMMENTED** [review]
I will review the TEASER aspect tomorrow, BESMod specific stuff looks fine, only minor comments.

File: BESMod/unitTests-dymola.log (hunk 1 above)
Comment (HvanderStok) [reply BESMod/unitTests-dymola.log:1]: This is added by the CI. Should I remove it and add it to gitignore?

**Review by FWuellhorst — APPROVED** [review]

**Pull request merged**